homeassistant/components/august/sensor.py

```diff
@@ -0,0 +1,148 @@
+"""Support for August sensors."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.sensor import DEVICE_CLASS_BATTERY
+from homeassistant.helpers.entity import Entity
+
+from . import DATA_AUGUST
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=5)
```

Comment:
We should leave this to 10.

Reply:
Will adjust way up with the upcoming PR (see previous comment)

Reply:
Part 1 of 2 of this is here : https://github.com/home-assistant/home-assistant/pull/32193

Reply:
Part 2 of 2 is here 
https://github.com/bdraco/home-assistant/tree/activity_stream_on_top_of_integration
Will squash and submit PR after https://github.com/home-assistant/home-assistant/pull/32193 merges